inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/StreamSourceEntity.java

```diff
@@ -47,6 +47,7 @@ public class StreamSourceEntity implements Serializable {
     private String serializationType;
     private String snapshot;
     private Date reportTime;
+    private String agentTimeZone;
```

Comment:
This is not the time zone of the agent, it is the time zone of the manager or sort